kDrive/UI/Controller/Files/Save File/SaveFileViewController.swift

```diff
@@ -161,6 +159,30 @@ class SaveFileViewController: UIViewController {
         )
     }
 
+    func getBestDirectory() -> File? {
+        if lastSelectedDirectory?.driveId == selectedDriveFileManager?.drive.id {
+            return lastSelectedDirectory
+        }
+
+        guard let selectedDriveFileManager else { return nil }
+
+        let myFilesDirectory = selectedDriveFileManager.database.fetchResults(ofType: File.self) { files in
+            files.filter(NSPredicate(format: "rawVisibility = %@", FileVisibility.isPrivateSpace.rawValue))
```

Comment:
Tiny detail, we can write this directly.
```suggestion
            files.filter("rawVisibility = %@", FileVisibility.isPrivateSpace.rawValue)
```